src/styles/global/elements.scss

```diff
@@ -16,7 +16,7 @@ button {
 
 html {
   position: relative;
-  font-size: ($base-font-size / $default-browser-font-size) * 100%;
+  font-size: ($root-font-size / 16px) * 100%;
```

Comment:
Because the `$root-font-size` variable still exists (see above), I’ve opted to keep this line.